up-socket-cpp/developer/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git
+from conan.tools.files import copy
+
+
+class UpClientSocket(ConanFile):
+    name = "up_client_socket"
+
+    # Optional metadata
+    license = "Apache-2.0 license"
+    author = "<Put your name here> <And your email here>"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "C++ up client socket library for testagent"
+    topics = ("automotive", "iot", "uprotocol", "messaging")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+            "shared": [True, False],
+            "fPIC": [True, False],
+            "fork": ["ANY"],
+            "commitish": ["ANY"] }
+
+    default_options = {
+            "shared": False,
+            "fPIC": True,
+            "fork": "eclipse-uprotocol/up-tck",
+            "commitish": "main"}
+
+    requires = "up-core-api/[~1.6, include_prerelease]", "spdlog/[~1.13]", "protobuf/[~3.21]", "up-cpp/[^1.0, include_prerelease]", "fmt/10.2.1", "openssl/1.1.1w"
```

Comment:
Does it really require openssl?

Reply:
Done, Something which remained from previous release